README.md

```diff
@@ -299,6 +299,7 @@ _Note that for the `producer` mode, the prebuild build phase and `xccc`, `xcld`,
 | `download_retries` | Number of retries for download requests | `0` | ⬜️ |
 | `upload_retries` | Number of retries for upload requests | `3` | ⬜️ |
 | `retry_delay` | Delay between retries in seconds | `10` | ⬜️ |
+| `max_concurrent_requests` | Maximum number of simultaneous requests. 0 means no limits | `0` | ⬜️ |
```

Comment:
The implementation seems to respect that only in `PUT` requests, `GET` is not limited by that property. Shouldn't we call it something like `max_concurrent_upload_requests`?

Reply:
That makes sense 👍 I would like to add this logic for GET as well, but I haven't found a place where we do multiple parallel get requests. Do we have such a place?

Reply:
We trigger multiple GETs in parallel for the thinning plugin postbuild consumer: 
https://github.com/spotify/XCRemoteCache/blob/master/Sources/XCRemoteCache/Commands/Plugins/Thinning/ThinningConsumerPostbuildPlugin.swift#L140-L142

That code downloads (also unzips and moves) plenty of .zip artifacts. It is managed by a single `URLSession` instance so that doesn't oversubscribe network resources.